### ISSUE TITLE
About the share weights?

### ISSUE THREAD

**Issue by tjulyz** [layumi/2016_person_re-ID#12]
Hi, @layumi 
Thanks a lot for your code.
There is a question which puzzles me. Are the two branches of Resnet in your code share the same weights? I find that you initialize the parameters by the pretrained model and change the learning rate of parameters in each branch. But I haven't found how to make the two branches share the same weights.
Can you help me that?
Thanks!

**Comment by layumi** [comment]
Hi @tjulyz 
If you go deep into the network, you can see the layers in both branch use the same 'weight name'. They use the same weight (params).

**Comment by tjulyz** [comment]
Thank you for providing the details. I will read it again.
